vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -265,6 +265,13 @@ struct ReactOptions {
       bool value) noexcept;
   static bool UseFastRefresh(winrt::Microsoft::ReactNative::IReactPropertyBag const &properties) noexcept;
 
+  JSIEngine GetJsiEngine() const noexcept;
+  void SetJsiEngine(JSIEngine value) noexcept;
```

Comment:
name these according to the property pattern:
```suggestion
  JSIEngine JsiEngine() const noexcept;
  void JsiEngine(JSIEngine value) noexcept;
```

Reply:
I'm updating the getters to remove the prefix .. but it looks like the existing pattern is to use "Set" prefix for setters ..